### DISCUSSION TITLE
Cannot show plug-in in OBS on Ubuntu

### DISCUSSION THREAD

**Discussion by minettsdapp** [WarmUpTill/SceneSwitcher#561] [Q&A]
I am trying since a few days to install this OBS plug-in on a Ubuntu machine. I don't know now if I'm going about the installation the wrong way.

I installed OBS like it's described on the [OBS homepage](https://obsproject.com/download). I downloaded the .zip file for the plugin from this [link](https://github.com/WarmUpTill/SceneSwitcher/releases/tag/1.19.1). Then I copied the two folders ("data" and "obs-plugins) into the .configure/obs-studio folder.

I expect to find advanced-scene-switcher under OBS in the "Tools" menu. However, this is not the case.

Has anybody out here any idea or an detailed documentation on how to install the plugin under Ubuntu?

Thanks.

**Comment by WarmUpTill** [comment]
Can you share an OBS log?
This should give hints what the issue might be.

**Reply by minettsdapp** [reply]
Here is the log:
[2022-10-09 12-20-41.txt](https://github.com/WarmUpTill/SceneSwitcher/files/9741377/2022-10-09.12-20-41.txt)

Unfortunately I can't find any reference here about the plug-in at all.

I have installed the OBS-Studio version 28.0.3 (64bit).

**Reply by WarmUpTill** [reply]
Indeed there is no indication at all of the plugin being loaded.

Are you sure `.configure/obs-studio` is correct?
I think it might be `~/.config/obs-studio/` instead.

Alternatively can you please try downloading [advanced-scene-switcher-linux-x86_64.deb](https://github.com/WarmUpTill/SceneSwitcher/releases/download/1.19.1/advanced-scene-switcher-linux-x86_64.deb) and running `sudo apt install /path/to/advanced-scene-switcher-linux-x86_64.deb`.

**Reply by minettsdapp** [reply]
I got it working now with the package installation method you mentioned as an alternative! So my issue is solved. Thank you!

I tried again using the "copy/paste" method and checking the path, but without success. I have no idea what I could do wrong.

Here an screenshot where I pasted the two folders in the `~/.config/obs-studio/` folder.
![Screenshot_20221009_212250](https://user-images.githubusercontent.com/41488428/194776338-80569d7d-315d-4be3-8370-4f722b74658f.png)
